### PR TITLE
[HLS-3409] - add helioszerolog

### PR DESCRIPTION
**Pull request by LironKS**

### CODE REVIEW

File: proxy-libs/helioszerolog/helioslogger.go [hunk 1]
```diff
@@ -0,0 +1,148 @@
...
+
+	"github.com/rs/zerolog"
+	"go.opentelemetry.io/otel/trace"
+
+	origin_zerolog "github.com/rs/zerolog"
+)
+const hsApiEndpoint = "https://app.gethelios.dev"
+
+func extractDataFromContext(otelContext context.Context, event *Event) {
+	if otelContext != nil {
```
Comment (nozik) [inline proxy-libs/helioszerolog/helioslogger.go:17]: Can otelContext be nil? It's not a pointer

File: proxy-libs/helioszerolog/helioslogger.go [hunk 2]
```diff
@@ -0,0 +1,148 @@
...
+	return l.WrappedLogger.WithLevel(level)
+}
+
+func (l *Logger) Log() *Event {
+	event := l.WrappedLogger.Log()
+	extractDataFromContext(*l.otelContext, event)
+	return event
+}
+
+// Print sends a log event using debug level and no extra field.
```
Comment (nozik) [inline proxy-libs/helioszerolog/helioslogger.go:96]: Is this out comment? If not, let's remove it

File: proxy-libs/helioszerolog/helioslogger.go [hunk 3]
```diff
@@ -0,0 +1,148 @@
...
+// Arguments are handled in the manner of fmt.Print.
+func (l *Logger) Print(v ...interface{}) {
+	l.WrappedLogger.Print(v...)
+}
+
+// Printf sends a log event using debug level and no extra field.
```
Comment (nozik) [inline proxy-libs/helioszerolog/helioslogger.go:102]: Same

**Review by nozik — CHANGES_REQUESTED** [review]
Looks good! See minor comments

File: proxy-libs/helioszerolog/helioslogger.go (hunk 1 above)
Comment (LironKS) [reply proxy-libs/helioszerolog/helioslogger.go:17]: When we are initializing the Logger we are setting the context to nil at first

Comment (nozik) [reply proxy-libs/helioszerolog/helioslogger.go:17]: Right, but the variable type in this case isn't a pointer to a context, it's a context

**Review by nozik — APPROVED** [review]

**Pull request merged**